docs/usage/misc/time.md

```diff
@@ -125,6 +186,16 @@ else
   logger.info("Not in time range")
 end
 
-(java.time.Month::NOVEMBER..java.time.Month::DECEMBER).cover?(Time.now)
-(java.time.Month::NOVEMBER..java.time.Month::DECEMBER).cover?(Date.today)
+java_import java.time.Month # This is already imported in OpenHAB 3.4
+
+# Compare against Month
+Time.now.between?(Month::NOVEMBER..Month::DECEMBER)
+Date.today.between?(Month::NOVEMBER..Month::DECEMBER)
+ZonedDateTime.now.between?(Month::NOVEMBER..Month::DECEMBER)
+
+# Compare against MonthDay
+ZonedDateTime.now.between?("05-01".."12-01")
```

Comment:
again. now that we've shown they all work, we should be pushing towards the shortest version - `Time.now..between?`

Reply:
good idea